consensus/polybft/polybft_config.go

```diff
@@ -197,3 +255,17 @@ func (m *Manifest) Save(manifestPath string) error {
 
 	return nil
 }
+
+// MakeKoskSignature creates KOSK signature which prevents rogue attack
+func MakeKoskSignature(
```

Comment:
nitpick: Should be `MakeKOSKSignature`

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1184/commits/96cf126ab95563ba348ca6ca1ffd515c1db49fc6